collections/Exeter_College/Exeter_College_MS_158.xml

```diff
@@ -77,7 +77,7 @@
                      <handNote>Written in a large number of gothic bookhands, some with documentary features such as the splitting of heads of ascenders. Punctuated by low point. Finucane, <title>Miracles and Pilgrims</title> <!--(above)-->, pl. 1, reproduces fols. 29<hi rend="superscript">v</hi>–30<hi rend="superscript">r</hi>, much reduced.</handNote>
                   </handDesc>
                   <decoDesc>
-                     <decoNote>Plain red initials, some ½-line and others with extenders; rubrics; red running titles, red paraphs.</decoNote>
+                     <decoNote type="plainInit">Plain red initials, some ½-line and others with extenders; rubrics; red running titles, red paraphs.</decoNote>
```

Comment:
I think this is my fault for not being completely clear int eh guidelines but "colInit" should be used for plain coloured initials (if that makes sense...) and "plainInit" for initials in the ink of the text. I suspect you can do a global change to replace these (or I can)